### ISSUE TITLE
Custom the Layers

### ISSUE THREAD

**Issue by ChrisNg02655332** [prevwong/craft.js#113]
Hi everyone,
I would like to say that’s this library is the one I want to get last year. Thank you for making it.
whether I can find the example about custom layers?
Thank you in advance.

**Comment by RaGreen** [comment]
Do you mean custom layers pane or?

**Comment by gcphost** [comment]
I would like to know how to customize the colors in the layers pane

**Comment by kbone1997** [comment]
still searching for this ans

**Comment by TarekFaysal** [comment]
Is there any way to create a custom UI for individual layer cards?

**Comment by superlc** [comment]
> Is there any way to create a custom UI for individual layer cards?

Yes, there is a prop (renderLayer) which can be used to custom the layer UI.

1. `<Layers renderLayer={LayersRenderer} expandRootOnLoad />`
2. Implement the UI Component LayerRenderer, you can reference the url [DefaultLayer.tsx](https://github.com/prevwong/craft.js/blob/main/packages/layers/src/layers/DefaultLayer/DefaultLayer.tsx)

**Comment by MrVibe** [comment]
I want to add a remove Layer option in the Layers panel, is that possible ?
